PluginCode/src/main/java/smr/cs/ualberta/libcomp/data/DependencyStatement.java

```diff
@@ -1,29 +1,14 @@
 package smr.cs.ualberta.libcomp.data;
 
-public class DependencyStatement {
+public class DependencyStatement extends Statement {
```

Comment:
does dependency mean both gradle and maven?

Reply:
yes. from what i understand, we use DependencyStatement for both gradle and maven